Compile error BOPAlgo_Tools TreatCompound

I am trying to compile the project, but getting an error: TreatCompound is not a member of BOPAlgo_Tools (see below for long message)

I have tried with:
pythonocc-core@7.4.0 and pythonocc-core@master
opencascade versions 7.4.0 / 7.4.0p1
swig 3.0.12 

My assumption is that the error is due to some refactoring, however I would have expected that any of the above versions should have worked. Can anybody help me to get pythonocc-core compile?


```
[ 49%] Building CXX object CMakeFiles/_BOPAlgo.dir/Unix/x86_64-Release-/BOPAlgoPYTHON_wrap.cxx.o
/home/common/software/python-pkgs/pythonocc-core/cmake-build/Unix/x86_64-Release-/BOPAlgoPYTHON_wrap.cxx: In function \u2018PyObject* _wrap_BOPAlgo_Tools_TreatCompound(PyObject*, PyObject*)\u2019:
/home/common/software/python-pkgs/pythonocc-core/cmake-build/Unix/x86_64-Release-/BOPAlgoPYTHON_wrap.cxx:10055:7: error: \u2018TreatCompound\u2019 is not a member of \u2018BOPAlgo_Tools\u2019
       BOPAlgo_Tools::TreatCompound((TopoDS_Shape const &)*arg1,*arg2,*arg3);
       ^~~~~~~~~~~~~
CMakeFiles/_BOPAlgo.dir/build.make:82: recipe for target 'CMakeFiles/_BOPAlgo.dir/Unix/x86_64-Release-/BOPAlgoPYTHON_wrap.cxx.o' failed
make[2]: *** [CMakeFiles/_BOPAlgo.dir/Unix/x86_64-Release-/BOPAlgoPYTHON_wrap.cxx.o] Error 1
CMakeFiles/Makefile2:9471: recipe for target 'CMakeFiles/_BOPAlgo.dir/all' failed
make[1]: *** [CMakeFiles/_BOPAlgo.dir/all] Error 2
Makefile:149: recipe for target 'all' failed
make: *** [all] Error 2

```

--- a/src/SWIG_files/wrapper/BOPAlgo.i
+++ b/src/SWIG_files/wrapper/BOPAlgo.i
@@ -1016,23 +1016,6 @@ None
 ") PerformCommonBlocks;
 		static void PerformCommonBlocks(const BOPDS_IndexedDataMapOfPaveBlockListOfInteger & theMBlocks, const opencascade::handle<NCollection_BaseAllocator> & theAllocator, BOPDS_PDS & pDS, const opencascade::handle<IntTools_Context> & theContext = opencascade::handle<IntTools_Context>());
 
-		/****************** TreatCompound ******************/
-		/**** md5 signature: b1cc9e0d4e7a7bd0544b9bc0f8d37159 ****/
-		%feature("compactdefaultargs") TreatCompound;
-		%feature("autodoc", "Collect in the output list recursively all non-compound subshapes of the first level of the given shape thes. if a shape presents in the map themfence it is skipped. all shapes put in the output are also added into themfence.
-
-Parameters
-----------
-theS: TopoDS_Shape
-theMFence: TopTools_MapOfShape
-theLS: TopTools_ListOfShape
-
-Returns
--------
-None
-") TreatCompound;
-		static void TreatCompound(const TopoDS_Shape & theS, TopTools_MapOfShape & theMFence, TopTools_ListOfShape & theLS);
-
 		/****************** WiresToFaces ******************/
 		/**** md5 signature: 16dc9996c77bddaa892446101a7cb4b5 ****/
 		%feature("compactdefaultargs") WiresToFaces;
@@ -1064,6 +1047,10 @@ bool
 
 	@methodnotwrapped
 	def FillMap(self):
+		pass
+
+	@methodnotwrapped
+	def TreatCompound(self):
 		pass
 	}
 };

--- a/src/SWIG_files/wrapper/BOPAlgo.pyi
+++ b/src/SWIG_files/wrapper/BOPAlgo.pyi
@@ -167,8 +167,6 @@ class BOPAlgo_Tools:
 	@staticmethod
 	def PerformCommonBlocks(theMBlocks: BOPDS_IndexedDataMapOfPaveBlockListOfInteger, theAllocator: NCollection_BaseAllocator, pDS: BOPDS_PDS, theContext: Optional[IntTools_Context] = IntTools_Context()) -> None: ...
 	@staticmethod
-	def TreatCompound(theS: TopoDS_Shape, theMFence: TopTools_MapOfShape, theLS: TopTools_ListOfShape) -> None: ...
-	@staticmethod
 	def WiresToFaces(theWires: TopoDS_Shape, theFaces: TopoDS_Shape, theAngTol: Optional[float] = 1e-8) -> bool: ...
 
 class BOPAlgo_WireEdgeSet:
@@ -481,7 +479,6 @@ BOPAlgo_Tools_FillInternals = BOPAlgo_Tools.FillInternals
 BOPAlgo_Tools_IntersectVertices = BOPAlgo_Tools.IntersectVertices
 BOPAlgo_Tools_PerformCommonBlocks = BOPAlgo_Tools.PerformCommonBlocks
 BOPAlgo_Tools_PerformCommonBlocks = BOPAlgo_Tools.PerformCommonBlocks
-BOPAlgo_Tools_TreatCompound = BOPAlgo_Tools.TreatCompound
 BOPAlgo_Tools_WiresToFaces = BOPAlgo_Tools.WiresToFaces
 BOPAlgo_MakePeriodic_ToDirectionID = BOPAlgo_MakePeriodic.ToDirectionID
 BOPAlgo_ShellSplitter_SplitBlock = BOPAlgo_ShellSplitter.SplitBlock

